src/aof.c

```diff
@@ -872,7 +872,7 @@ int aofRewriteLimited(void) {
     }
 
     next_rewrite_time = server.unixtime + next_delay_minutes * 60;
-    serverLog(LL_WARNING,
+    serverLog(LL_NOTICE,
```

Comment:
oran: i suppose it's ok to make this a NOTICE since the actual error is probably logged at WARNING

Reply:
I think we can leave it a warning, it does provide additional information of warning severity.

Reply:
leave it back to warning